ext/storage_sources/dir_store/dir_store.c

```diff
@@ -200,6 +203,8 @@ dir_store_configure(DIR_STORE *dir_store, WT_CONFIG_ARG *config)
     if ((ret = dir_store_configure_int(
            dir_store, config, "force_delay", &dir_store->force_delay)) != 0)
         return (ret);
+    if ((ret = dir_store_configure_int(dir_store, config, "error_ms", &dir_store->error_ms)) != 0)
+        return (ret);
```

Comment:
Move this code to alphabetize when we process this.

Reply:
Fixed in 0e6b43406c158379072274241f2c15de26c317b5